[INFO              ] Logger: Record log in C:\Home\Gazooli\.kivy\logs\kivy_15-03-09_1.txt
[INFO              ] Kivy v1.8.0
[INFO              ] Factory: 157 symbols loaded
[DEBUG             ] Cache: register <kv.lang> with limit=None, timeout=Nones
[DEBUG             ] Cache: register <kv.image> with limit=None, timeout=60s
[DEBUG             ] Cache: register <kv.atlas> with limit=None, timeout=Nones
[INFO              ] Image: Providers: img_tex, img_dds, img_pygame, img_gif (img_pil ignored)
[DEBUG             ] Cache: register <kv.texture> with limit=1000, timeout=60s
[DEBUG             ] Cache: register <kv.shader> with limit=1000, timeout=3600s
[DEBUG             ] App: Loading kv <C:/Users/wade7892/PycharmProjects/untitled\mypaint.kv>
[DEBUG             ] App: kv <C:/Users/wade7892/PycharmProjects/untitled\mypaint.kv> not found
[CRITICAL          ] App.root must be an _instance_ of Widget
[WARNING           ] stderr: Traceback (most recent call last):
[WARNING           ] stderr:   File "C:/Users/wade7892/PycharmProjects/untitled/test.py", line 15, in <module>
[WARNING           ] stderr:     MyPaintApp().run()
[WARNING           ] stderr:   File "C:\Python34\lib\site-packages\kivy\app.py", line 772, in run
[WARNING           ] stderr:     raise Exception('Invalid instance in App.root')
[WARNING           ] stderr: Exception: Invalid instance in App.root
